src/services/amiibo.services.mjs

```diff
@@ -0,0 +1,42 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "https://www.amiiboapi.com/api",
+});
+
+const MAX_CHARS = 778;
+
+function randomChar(maxChar) {
```

Comment:
Wenn du die Funktion extrahierst, gib ihr vielleicht nen neutraleren namen, der wiederspiegelt, dass es sich uim ne Funktion handelt, die nen random Integer zwischen 0 und X returnt.

Reply:
Habe ich extrahiert in helper/utils.helper -> randomNumberInLimit(limit)